weld/ast.rs

```diff
@@ -523,21 +522,22 @@ impl<T: TypeBounds> Expr<T> {
 
     /// Recursively transforms an expression in place by running a function on it and optionally replacing it with another expression.
     pub fn transform_and_continue<F>(&mut self, func: &mut F)
-        where F: FnMut(&mut Expr<T>) -> Option<(Expr<T>, bool)>
+        where F: FnMut(&mut Expr<T>) -> (Option<Expr<T>>, bool)
```

Comment:
What's the motivation for this change? Is the bool about whether to continue recursing down or something?